pkg/application/application.go

```diff
@@ -346,6 +348,10 @@ func kustCreateFiles(app *kustApp, repofs fs.FS, projectName string) error {
 	}
 
 	configPath := repofs.Join(overlayPath, "config.json")
```

Comment:
doesn't the configPath always in the main repo?

Reply:
yes, that is why it is always being written to `repofs`

Reply:
but this path is missing "overlays" no?

Reply:
if the repofs == appsfs
  we put the config.json in `apps/<appName>/overlays/<projectName>/config.json`
else // repofs != appsfs
  we put the config.json in `apps/<appName>/<projectName>/config.json`

there is no need for an `overlays` folder if there is no matching `base` folder in the repofs (I think)

Reply:
hmm, do we check if we try to add an application with the same name on the main repo (when the base was installed previously on some apps repo)?